### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Guillaume Fragnière, Dylan Langumier & Killian Viquerat

### DISCUSSION THREAD

**Discussion by GFragniere** [heig-vd-dai-course/heig-vd-dai-course#533] [Show and tell]
For this project, we decided to make a utilitary tool for the game: The Binding of Isaac.

This application will have the next functionnalities: 
- get all the information for an item or trinket by giving its ID
- get all the items from a specific item pool (treasure room, secret room, Devil's room, ...)
- get all the items of a specific quality tier
- use the "spindown" functionnality on any item (basically gets the item that has the ID prior to the given item)
- a random item generator (equivalent to the active item D6 inside the game) and add the ability to specify a pool of item
- add items inside the database with the pool, effect and quality
- update an item by modifying informations

**Comment by GeraudSilvestri** [comment]
Good for me

**Comment by maellemelody** [comment]
Here's the repository https://github.com/Dylan-Langumier/DAI-Practical-Work-3

**Comment by maellemelody** [comment]
Here's the reddition tag: https://github.com/Dylan-Langumier/DAI-Practical-Work-3/tree/v0.1.0-beta

**Comment by ludelafo** [comment]
Bonjour @Dylan-Langumier, @GFragniere et @Killian-Viquerat,

J'essaie d'accéder à votre serveur sous le nom de domaine dl-dai-heig.duckdns.org et le serveur ne semble pas répondre, ni par SSH ni par HTTP. Est-ce qu'il est allumé ?

Merci d'avance et bonne journée,
Ludovic

**Reply by maellemelody** [reply]
Bonjour @ludelafo 
Mes excuses, j'ai éteint le serveur par habitude hier après-midi étant donné qu'il n'était pas utilisé. Je viens de le relancer !

Bonne journée,
Dylan

**Reply by ludelafo** [reply]
Merci beaucoup !
